test/unit_tests/models/test_models.py

```diff
@@ -120,8 +121,65 @@ def test_eegitnet(input_sizes):
     check_forward_pass(model, input_sizes,)
 
 
-@pytest.mark.parametrize('n_channels,sfreq,n_classes,input_size_s',
-                         [(20, 128, 5, 30), (10, 256, 4, 20), (1, 64, 2, 30)])
+def test_eeginception(input_sizes):
```

Comment:
this should be deleted or what?

Reply:
Yes, I changed the way of taking the test and I ended up forgetting this old test.